devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -300,6 +354,87 @@ private command(physicalgraph.zwave.Command cmd) {
 	}
 }
 
+def getConfigurationCommands() {
+	log.debug "getConfigurationCommands"
+	def result = []
+	if (isEverspringSP815()) {
+		if (!state.temperatureAndHumidityReport) state.temperatureAndHumidityReport = getEverspringDefaults[1]
+		if (!state.retriggerIntervalSettings) state.retriggerIntervalSettings = getEverspringDefaults[2]
+
+		Integer temperatureAndHumidityReport = (settings.temperatureAndHumidityReport as Integer) ?: everspringDefaults[1]
+		Integer retriggerIntervalSettings = (settings.retriggerIntervalSettings as Integer) ?: everspringDefaults[2]
+
+		if (!isConfigured() || (temperatureAndHumidityReport != state.temperatureAndHumidityReport || retriggerIntervalSettings != state.retriggerIntervalSettings)) {
+			state.configured = false // this flag needs to be set to false when settings are changed (and the device was initially configured before)
+			if (!state.temperatureConfigured || temperatureAndHumidityReport != state.temperatureAndHumidityReport) {
+				state.temperatureConfigured = false
+				result << zwave.configurationV2.configurationSet(parameterNumber: 1, size: 2, scaledConfigurationValue: temperatureAndHumidityReport)
+				result << zwave.configurationV2.configurationGet(parameterNumber: 1)
+			}
+			if (!state.intervalConfigured || retriggerIntervalSettings != state.retriggerIntervalSettings) {
+				state.intervalConfigured = false
+				result << zwave.configurationV2.configurationSet(parameterNumber: 2, size: 2, scaledConfigurationValue: retriggerIntervalSettings)
+				result << zwave.configurationV2.configurationGet(parameterNumber: 2)
+			}
+		}
+	} else if (isEverspringSP817()) {
+		if (!state.retriggerIntervalSettings) state.retriggerIntervalSettings = everspringDefaults[2]
+		Integer retriggerIntervalSettings = (settings.retriggerIntervalSettings as Integer) ?: everspringDefaults[2]
+		if (!isConfigured() || (retriggerIntervalSettings != state.retriggerIntervalSettings)) {
```

Comment:
shouldn't you set `state.configured` here?

Reply:
yes, it's done now